### PR TITLE
Noe/crash on load

### PR DESCRIPTION
**Pull request by nmalzieu**
Seems to fix the crash on wallet click (version 237 on testflight works for me)
Also fixes local building that was failing on my computer

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,8 +23,14 @@
   },
   "dependencies": {
     "@azure/core-asynciterator-polyfill": "^1.0.2",
+    "@babel/core": "^7.25.2",
     "@babel/plugin-proposal-decorators": "^7.20.7",
     "@babel/plugin-proposal-export-namespace-from": "^7.18.9",
+    "@babel/plugin-transform-class-properties": "^7.24.7",
```
Comment (alexrisch) [inline package.json:29]: I don't think babel should be required in dependencies rather than devDependencies

Is there a reason these are being moved?

**Comment by alexrisch** [comment]
Rebased with latest main

File: package.json (hunk 1 above)
Comment (nmalzieu) [reply package.json:29]: Yes building was failing due to missing dependencies, I'm not sure why, and followed that comment
https://github.com/facebook/create-react-app/issues/13189#issuecomment-1686243701

Comment (nmalzieu) [reply package.json:29]: I agree this is not ideal but wonder if there is any risk?

Comment (alexrisch) [reply package.json:29]: Bundle size would be the biggest concern, and if this workaround is masking a different issue

**Review by alexrisch — APPROVED** [review]

**Comment by alexrisch** [comment]
Made an issue to track in the future, but would like to include these changes in the next build

**Pull request merged**